ORA.Tracker/Routes/Members.cs

```diff
@@ -58,13 +58,38 @@ protected override byte[] post(HttpListenerRequest request, HttpListenerResponse
 
             var c = ClusterDatabase.Get(urlParams["id"])
                 ?? throw new HttpListenerException(404, invalidClusterId);
-            if (TokenManager.Instance.GetIdFromToken(token) != c.owner)
+            string userId = TokenManager.Instance.GetIdFromToken(token);
+            if (userId != c.owner || !c.admins.Contains(userId))
```

Comment:
??? logical or should be a logical and here

Reply:
Yes obviously, my brain lagged :D